tests/Aspire.Confluent.Kafka.Tests/KafkaContainerFixture.cs

```diff
@@ -0,0 +1,111 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text;
+using System.Text.RegularExpressions;
+using Aspire.Components.Common.Tests;
+using Docker.DotNet.Models;
+using DotNet.Testcontainers.Builders;
+using DotNet.Testcontainers.Configurations;
+using Testcontainers.Kafka;
+using Xunit;
+
+namespace Aspire.Confluent.Kafka.Tests;
+
+public partial class KafkaContainerFixture : IAsyncLifetime
+{
+    private sealed partial class ConfluentLocalKafkaBuilder : ContainerBuilder<ConfluentLocalKafkaBuilder, KafkaContainer, KafkaConfiguration>
+    {
+        public const string KafkaImage = "confluentinc/confluent-local:7.7.1";
+
+        public const ushort KafkaPort = 9092;
+
+        public const string StartupScriptFilePath = "/testcontainers.sh";
+
+        public ConfluentLocalKafkaBuilder()
+            : this(new KafkaConfiguration())
+        {
+            DockerResourceConfiguration = Init().DockerResourceConfiguration;
+        }
+
+        private ConfluentLocalKafkaBuilder(KafkaConfiguration resourceConfiguration)
+            : base(resourceConfiguration)
+        {
+            DockerResourceConfiguration = resourceConfiguration;
+        }
+
+        protected override KafkaConfiguration DockerResourceConfiguration { get; }
+
+        public override KafkaContainer Build()
+        {
+            Validate();
+            return new KafkaContainer(DockerResourceConfiguration);
+        }
+
+        protected override ConfluentLocalKafkaBuilder Init()
+        {
+            return base.Init()
+                .WithImage(KafkaImage)
+                .WithPortBinding(KafkaPort, true)
```

Comment:
Do we need to copy all this from test containers? Can't we just override the Image like we do elsewhere?

https://github.com/dotnet/aspire/blob/39912824d33de54be24baf842bf9228a6a84ac33/tests/Aspire.RabbitMQ.Client.Tests/RabbitMQContainerFixture.cs#L36-L37

Reply:
No we can't, that's why i first used the kafka 6.x images. 

Please notice that client side packages should not be bound to a specific version of kafka. Confluent.Kafka producer and consumer can be used with other queue system that support the kafka protocol like redpanda or Azure Event Hub.